lcio/trackerrawdata.go

```diff
@@ -19,10 +19,11 @@ type TrackerRawDataContainer struct {
 }
 
 type TrackerRawData struct {
-	CellID0 int32
-	CellID1 int32
-	Time    int32
-	ADCs    []uint16
+	CellID0    int32
+	CellID1    int32
+	Time       int32
+	nADCValues int32
+	ADCValues  []uint16
```

Comment:
There is a reason, though not a great one.  I only changed it since I was there, and I found "ADCs" slightly misleading.  To me it is suggestive that the contents of the slice are identifiers for _what_ ADCs were hit.  In my opinion, "ADCValues" makes it easier to understand what the slice represents.